components/scream/cime_config/buildnml

```diff
@@ -26,6 +26,50 @@ import yaml
 
 logger = logging.getLogger(__name__)
 
+###############################################################################
+def load_comments(filepath):
```

Comment:
Can you add a comment to these two functions explaining what they achieve and how they can be useful? For the sake of who will come next to maintain them. I'm having a hard time parsing them.